solidity/test/BancorNetwork.js

```diff
@@ -450,13 +452,13 @@ contract('BancorNetwork', accounts => {
         });
 
         it('should throw when calling convertFor with invalid path', async () => {
-            let invalidPath = [utils.zeroAddress, smartToken1.address];
+            let invalidPath = [ETH_RESERVE, smartToken1.address];
             await utils.catchRevert(bancorNetwork.convertFor(invalidPath, 10000, 1, accounts[1], { value: 10000 }));
         });
 
         it('should throw when calling convertFor with invalid long path', async () => {
             let longBuyPath = [];
-            for (let i = 0; i < 100; ++i)
+            for (let i = 0; i < 101; ++i)
```

Comment:
why change from 100 to 101?

Reply:
@yudilevi : Did some cleanup around this; please see updated code.

Reply:
ok